Crash on compression

## Exception
* User Action: UI Error
* Request: Application crash
* Version: 3.5.2
* OS: Linux Android 11 - 30
* Device: raphaelin
* Model: Redmi K20 Pro
* Product: evolution_raphael
<details><summary><b>Crash log </b></summary><p>


java.lang.RuntimeException: Unable to start service com.amaze.filemanager.asynchronous.services.ExtractService@7593294 with null: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Intent.getStringExtra(java.lang.String)' on a null object reference
 at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4338)
 at android.app.ActivityThread.access$1800(ActivityThread.java:237)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1951)
 at android.os.Handler.dispatchMessage(Handler.java:106)
 at android.os.Looper.loop(Looper.java:223)
 at android.app.ActivityThread.main(ActivityThread.java:7661)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:594)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Intent.getStringExtra(java.lang.String)' on a null object reference
 at com.amaze.filemanager.asynchronous.services.ExtractService.onStartCommand(ExtractService.java:96)
 at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4320)
 ... 8 more


</details>
<hr>

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java
@@ -166,7 +166,7 @@ public class CopyService extends AbstractProgressiveService {
     new DoInBackground(isRootExplorer).execute(b);
 
     // If we get killed, after returning from here, restart
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   @Override

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/DecryptService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/DecryptService.java
@@ -146,7 +146,7 @@ public class DecryptService extends AbstractProgressiveService {
     super.progressHalted();
     new DecryptService.BackgroundTask().execute();
 
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   class BackgroundTask extends AsyncTask<Void, Void, Void> {

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/EncryptService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/EncryptService.java
@@ -142,7 +142,7 @@ public class EncryptService extends AbstractProgressiveService {
     super.progressHalted();
     new BackgroundTask().execute();
 
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   @Override

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/ExtractService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/ExtractService.java
@@ -151,7 +151,7 @@ public class ExtractService extends AbstractProgressiveService {
     super.progressHalted();
     new DoWork(this, progressHandler, file, extractPath, entries).execute();
 
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   @Override

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/ZipService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/ZipService.java
@@ -151,7 +151,7 @@ public class ZipService extends AbstractProgressiveService {
     asyncTask = new CompressAsyncTask(this, baseFiles, mZipPath);
     asyncTask.execute();
     // If we get killed, after returning from here, restart
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   @Override

--- a/app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.java
+++ b/app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.java
@@ -147,7 +147,7 @@ public class FtpService extends Service implements Runnable {
 
     startForeground(NotificationConstants.FTP_ID, notification);
 
-    return START_STICKY;
+    return START_NOT_STICKY;
   }
 
   @Override

